src/main/java/edu/kis/vh/nursery/list/IntArrayStack.java

```diff
@@ -9,35 +9,41 @@
     private final int[] NUMBERS = new int[STACK_CAPACITY];
     private int total = EMPTY_STACK_VALUE;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)